README.md

```diff
@@ -1 +1,42 @@
-# Module-1---Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+> This project was done using Html and CSS
+
+
+## Done With
+ 
+ - HTML
+ - CSS
+ - linters
+
+## Authors
+
+👤 *Author1*
+
+- GitHub: [@githubhandle](https://github.com/Rudy-Menekam)
+- Twitter: [@githubhandle](https://twitter.com/MenekamR)
+- LinkedIn: [@githubhandle](https://www.linkedin.com/in/menekam-rudy/)
```

Comment:
- [ OPTIONAL] Kindly make sure, instead of @githubhandle, you put the proper handle for section to make it look more professional